app/chitter_app.rb

```diff
@@ -0,0 +1,27 @@
+require 'sinatra/base'
+require './lib/user'
+require './lib/peep'
+require './db/database_connection_setup'
+
+class ChitterApp < Sinatra::Base
+  enable :sessions
+
+  get '/' do
+    redirect '/chitter/index'
+  end
+
+  get '/chitter/home' do
```

Comment:
Do we need to call this chitter/home, instead of just '/' which represents the home route?